website/docs/sql/scalar_func/datetime.md

```diff
@@ -550,32 +550,32 @@ fiscal_quarter
 fiscal_year
 ```
 
-The following fiscal calendar options are available:
+The following fiscal calendar options need to be set:
 
-`start_month => <value>`
-:   The month in which the fiscal year starts. If the fiscal year starts
-    in February, `<value>` should be `2`. The default value is `1` (January).
+`fiscal_year_start_month => <value>`
+:   The month in which the fiscal year starts (1 - 12). If the fiscal year starts
+    in February, `<value>` should be `2`.
 
 `first_day_of_week => <value>`
-:   The first day of the fiscal week. If the fiscal year starts with
+:   The first day of the fiscal week (1 - 7). If the fiscal year starts with
     January, and `<value>` is `1` (Monday), `fiscal_week` on date
     (timestamp) `2023-01-02` will return `2`, since `2023-01-01` is a
-    Sunday. The default value is `1` (Monday).
+    Sunday. This option is only required for `fiscal_week` field.
```

Comment:
what about normal non-fiscal weeks?
Is it still taken into account also for non-fiscal weeks, even if not required?